MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+include AUTHORS.rst
```

Comment:
To reduce some bloat and things to remember to update, do we really need a `MANIFEST.in`?